readme.txt

```diff
@@ -191,6 +191,9 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+= [TBD] TBD =
+* Tweak - Re-styled Inactive RSVP Block in Block Editor
```

Comment:
When doing modifications of this size, lets be more descriptive in what we did. What was the purpose here.

Reply:
That is all good. Lets go with that.